### PR TITLE
Add flag strict to signatureImport

### PR DESCRIPTION
**Pull request by fanatid**
Resolve issue #83

### CODE REVIEW

**Comment by fanatid** [comment]
Does anybody reviewed this PR?

**Comment by jprichardson** [comment]
utACK. Should `strict` default to `true`?

**Comment by fanatid** [comment]
@jprichardson right now `strict` is `true` by default

**Comment by fanatid** [comment]
I planning merge this in next 12 hours if there is no objection.

File: lib/der.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+var ZERO_BUFFER_32 = new Buffer([
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+])
+
+exports.privateKeyExport = function (privateKey, publicKey, compressed) {
+  var result = new Buffer(compressed ? EC_PRIVKEY_EXPORT_DER_COMPRESSED : EC_PRIVKEY_EXPORT_DER_UNCOMPRESSED)
+  privateKey.copy(result, compressed ? 8 : 9)
+  publicKey.copy(result, compressed ? 181 : 214)
+  return result
```
Comment (dcousens) [inline lib/der.js:61]: Won't this leave the private key in EC_PRIVKEY_EXPORT_DER_UNCOMPRESSED until the next export?

Comment (fanatid) [inline lib/der.js:61]: Interesting question!

``` js
var orig = new Buffer([0x00, 0x01, 0x02, 0x03])
var result = new Buffer(orig)
new Buffer([0x04, 0x05]).copy(result, 0)
console.log(orig, result) // <Buffer 00 01 02 03> <Buffer 04 05 02 03>
```

as I assumed, answer is no

Comment (dcousens) [inline lib/der.js:61]: My bad,  of course it won't.  Sweet. LGTM

**Pull request merged**